app/assets/javascripts/widgets/timeago.js

```diff
@@ -26,6 +26,14 @@
           }
         });
       }
+
+      $.timeago.settings.autoDispose = false;
+
+      var initTimeAgo = function() {
+        $("time[data-time-ago]").timeago();
+      };
+      $(initTimeAgo);
+      $(document).on("turbolinks:load page:load ajax:success", initTimeAgo);
```

Comment:
This is racy. Basically, the handler for the ajax success here races with [the one after loading the comments](https://github.com/diaspora/diaspora/blob/caaea50cf61d1c54e876e16f4464af087dbf1750/app/assets/javascripts/mobile/mobile_comments.js#L106), basically assuming that the latter runs first so that the `initTimeAgo()` call here is able to find the new html.

One possible workaround would be calling `$("time[data-time-ago]").timeago();` after the `insertAfter` in the comment ajax success handler to make sure it gets called *after* the html was added or to add and dispatch an arbitrary event (`diaspora:pagecontentschanged` for example) instead, making sure we dispatch that every time when we are done adding stuff to the document.